pkg/controller/config/config_controller.go

```diff
@@ -198,9 +198,12 @@ func (r *ReconcileConfig) Reconcile(ctx context.Context, request reconcile.Reque
 	r.cacheManager.ExcludeProcesses(newExcluder)
 	configSourceKey := aggregator.Key{Source: "config", ID: request.NamespacedName.String()}
 	if err := r.cacheManager.UpsertSource(ctx, configSourceKey, gvksToSync); err != nil {
+		r.tracker.For(instance.GroupVersionKind()).TryCancelExpect(instance)
```

Comment:
`instance.GroupVersionKind()` may not actually return anything. I'd verify that it does.

If not, just hardcode the config GVK.

Reply:
from my testing, this works as expected. I can hard code the GVK if that's a better pattern but I see this being used in other controllers too.

I can do what I did for the syncset controller for now, for consistency.